tests/PHPCurlClass/PHPCurlClassTest.php

```diff
@@ -995,36 +997,6 @@ public function testJSONRequest()
             $this->assertEquals($expected_response, $test->server('post_json', 'POST', json_encode($data)));
 
             foreach (array(
-                'Content-Type',
```

Comment:
`CaseInsensitiveArray` has it's own tests, re-testing it in this way serves no further purpose.  A false-positive in one such test (generic POST, JSON or XML) would result in the others failing, so I removed this to conserve time (minimally).